README.md

```diff
@@ -2,44 +2,44 @@
 
 <h1 align="center"><img src="https://yagpdb.xyz/static/img/logo_y.png" height=32px width=32px></img>&nbspYAGPDB Custom Commands</h1>
 
-> <p align="center">A curated collection of CCs for your server</p>
+> <p align="center">A curated collection of custom commands for your server</p>
 
 <p align="center">
 	<i>
-		If you are not planning on contributing, but rather simply viewing custom commands, please use our <a href="https://yagpdb-cc.github.io">website</a> instead, which displays all custom commands available here in a user-friendly manner!
+		If you are looking to browse through our collection of custom commands as opposed to contributing, please refer to our <a href="https://yagpdb-cc.github.io">website</a> instead, which displays all custom commands available in a user-friendly manner.
```

Comment:
```suggestion
		If you are looking to browse through our collection of custom commands, please refer to our <a href="https://yagpdb-cc.github.io">website</a> instead, which displays all custom commands available in a user-friendly manner.
```

I think "just browsing" implies "not looking to contribute", so this feels a bit redundant.